### PR TITLE
outline: 0.69.2 -> 0.70.2

### PR DESCRIPTION
**Pull request by xanderio**
###### Description of changes
https://github.com/outline/outline/releases/tag/v0.70.0

Starting with outline 0.70.0 DB migrations are run via `umzug` and no longer by calling the `sequelize` command at runtime, with this upstream change we can remove our patch and some code from the NixOS unit.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by yrd** [comment]
Awesome, thanks!
Mind bumping to [0.70.2](https://github.com/outline/outline/releases/tag/v0.70.2) while you're at it?

**Review by yrd — CHANGES_REQUESTED** [review]
Looks good; just deployed your branch on my local instance and everything seems to work 👍🏻 

And could you please change the path in this example here to `plugins/oidc/server/auth/oidc.ts`? It seems stuff has been moved around since https://github.com/outline/outline/commit/21a1257d0652cb3d043c99b13db5884f3206708d
https://github.com/NixOS/nixpkgs/blob/451adc94255484eb751cac1d713c5f4bf6fee324/nixos/modules/services/web-apps/outline.nix#L32

File: nixos/modules/services/web-apps/outline.nix [hunk 1]
```diff
@@ -51,15 +55,6 @@ in
       '';
     };
 
-    sequelizeArguments = lib.mkOption {
```
Comment (cab404) [inline nixos/modules/services/web-apps/outline.nix:54]: Doesn't that also require a changelog note on a breaking change?

**Review by cab404 — COMMENTED** [review]
Thank you!

**Review by cab404 — APPROVED** [review]
lgtm!

**Review by yrd — APPROVED** [review]
Nice, LGTM!

**Comment by RaitoBezarius** [comment]
Would you consider backports for NixOS 23.05 given that current Outline is using Node.js EOL?

**Pull request merged**